bugwarrior/command.py

```diff
@@ -67,6 +74,8 @@ def pull(dry_run, flavor, interactive, debug):
         )
     except RuntimeError as e:
         log.critical("Aborted (%s)" % e)
+    except SystemExit:
+        pass
```

Comment:
Ooops. This was as otherwise the generic catch all exceptions triggers and you get a traceback which clouds the helpful error message.